### PR TITLE
Text review

### PR DESCRIPTION
**Pull request by Deren-Liao**
Update texts from text review feedback.

### CODE REVIEW

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 1]
```diff
@@ -1512,40 +1512,39 @@
...
     <comment>Caption for source line tooltip back button.</comment>
   </data>
   <data name="LogsViewerVersionInfoMissingMessage" xml:space="preserve">
-    <value>The log entry does not contain valid assembly name and/or assembly version.</value>
+    <value>The log entry does not contain valid assembly name and/or assembly version</value>
     <comment>The warning message when a log entry does not contain valid version information.</comment>
   </data>
   <data name="LogsViewerVersionMismatchPrompt" xml:space="preserve">
-    <value>Version missmatch. \r\nThe current project {0} version is {1},\r\nOpen the project of version {2} to properly locate the logging writter location.</value>
+    <value>The current project {0} version is {1},\r\nOpen the project of version {2} to properly locate the logging writer location.</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: I'm a little confused by this text.  What is a "project version" in this context?

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 2]
```diff
@@ -1512,40 +1512,39 @@
...
     <comment>Prompt when the opened project version differs than the version information of the log entry.</comment>
   </data>
   <data name="LogsViewerVersionMissmatchAskIgnoreMessage" xml:space="preserve">
-    <value>"Do you want to continue anyway?"</value>
+    <value>Do you want to continue with current version?</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: "with the"

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx [hunk 3]
```diff
@@ -2073,35 +2072,43 @@ Open the project to navigate to the logger method location.</value>
...
   <data name="LogsViewerSourceLinkCaptionFormat" xml:space="preserve">
     <value>({0}:{1})</value>
     <comment>The source file link caption string format.  {0} is file name part, {1} is line number.</comment>
   </data>
+  <data name="SourceVersionUtilsUnalbeFindFileTitle" xml:space="preserve">
+    <value>Unable to find source file</value>
+    <comment>The title for error dialog when Logs View or Error Reporting is not able to find a source file from the current solution.</comment>
+  </data>
+  <data name="SourceVersionUtilsVersionMismatchTitle" xml:space="preserve">
+    <value>Versions don't match</value>
```
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Suggestion: Verion mismatch

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 1 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: The log items have the assembly version number.  The project assembly version might be clearer?

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 3 above)
Comment (Deren-Liao) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Text review firstly suggested Version mismatch and then changed to Versions don't match.

File: GoogleCloudExtension/GoogleCloudExtension/Resources.resx (hunk 1 above)
Comment (iantalarico) [inline GoogleCloudExtension/GoogleCloudExtension/Resources.resx]: Yea I think that would be more clear.  The message is hard to parse without the context (which may be shown when this is displayed).

**Comment by Deren-Liao** [comment]
Modified, please take a look again.
